src/events/2023jodoseminar.md

```diff
@@ -0,0 +1,256 @@
+---
+layout: page
+title: "2023 Jodo Seminar"
+permalink: "/events/2023jodoseminar/"
+lang: "en"
+---
+
+<style>
+  section:not(:first-of-type) {
+    margin-top: 3rem;
+  }
+  a.registration {
+    font-weight: bold;
+    font-size: larger;
+  }
+  @media screen and (min-width: 600px) {
+    a.registration {
+      width: 50%;
+      margin: auto;
+    }
+  }
+  .list, .schedule ol {
+    list-style: disc;
+    margin: 1.5rem 0 1.5rem 3rem;
+  }
+  .gmap {
+    display: block;
+    position: relative;
+    margin: 1rem auto;
+  }
+
+  @media screen and (max-width: 599px) {
+    .gmap {
+      width: 90% !important;
+    }
+  }
+
+  .schedule > li {
+    margin: 1rem 0;
+  }
+  .schedule span {
+    font-weight: bold;
+  }
+
+</style>
+
+<h1>2023 Jodo Seminar</h1>
+
+<section>
+  <h2>Welcome to registration for the 2023 Hoshu Dojo Jodo Camp!</h2>
+
+  <p>This year will be Goto-Sensei's 10th trip to the Americas, and to honour the special anniversary we will be doing a
+    gasshuku-style event - an all-inclusive camp where we train, stay, eat, and socialize together.</p>
+
+  <p>This year's camp will be held at Seattle University in Seattle, WA, from Thursday July 6th through Sunday July 9th, with
+    training across two and a half days from Friday through Sunday. Both room and board will also be provided by Seattle U,
+    with the exception of the "formal" camp dinner on Saturday night held at Olmstead Restaurant. There will be several options of
+    rooms available with differing price ranges. While staying at Seattle U is not required, it is highly encouraged. Food is
+    available at the university even if you are not staying there - but will need to be pre-registered.</p>
+
+  <p>Read on for more information about the camp, including the seminar schedule, accommodiation, meals, and parking.</p>
+
+  <p>We look forward to seeing you in July!</p>
+</section>
+
+<section>
+  <p><a class="c-button registration"
+        href="https://docs.google.com/forms/d/e/1FAIpQLScAEBHP5Z_UEyR4qcUPEaxPzPXljhoJwsyzIooRw4cVCvXNAg/viewform"
+        target="_blank"> Click
+    here to
+    register</a></p>
+
+  <p>Since this year's camp involves coordination with Seattle University, deadlines for registration and payment are
+    <b>Sunday April 30th</b> for Early Bird pricing, and <b>Friday June 9th</b> for regular pricing.</p>
```

Comment:
I would suggest that the explanation for why there's a deadline is secondary to the point about there being a deadline. Something to the effect of...

```suggestion
  <h2>Registration deadline</h2>
  <p><strong>Early bird deadline:</strong> Sunday April 30</p>
  <p><strong>Final deadline:</strong> Friday June 9th</p>
  <p>Why is there a deadline? Because this year's camp involves coordination with Seattle University.</p>
```

Feels like there could be more of a reason there, but anyway the main thing is ensuring the dates are front and center.

Reply:
Yeah. I'll just list the deadlines, and leave the explanation out.